### PR TITLE
Fix formatting, use impulse instead of force

### PR DESCRIPTION
**Pull request by darabos**
Plus the bullet ends up inside the boxes and is not on top of their heads at the end. 😄 

I looked a bit into a particle system... I think it's doable, but the black balls are not too bad either. Maybe later.

### CODE REVIEW

**Comment by darabos** [comment]
(Sorry, the formatting was my fault.)

File: src/App.js [hunk 1]
```diff
@@ -124,30 +120,27 @@ function LoyaltyBar({max, current, change}) {
...
+    const src = sourceHero.position;
+    const dst = targetHero.position;
+    mesh.current && mesh.current.position.lerpVectors(src, dst, phase);
     if (props.turnClock.time === turnFrames && props.attackTurn === 0) {
-      const force = new THREE.Vector3(tX - sX, tY - sY, 0);
-      force.normalize().multiplyScalar(100);
+      const force = dst.clone().sub(src).normalize().multiplyScalar(5);
       const localForce = targetHero.worldToLocal(force);
-      targetHero.physicsApi.applyLocalForce(localForce.toArray(), [0, 0, 1]);
+      targetHero.physicsApi.applyLocalImpulse(localForce.toArray(), [0, 0, 1]);
```
Comment (xandrew) [inline src/App.js:143]: Why did you change this? I think force is better because then heavier heros are affected less. What is the benefit of impulse?
